.eslintrc.yml

```
@@ -0,0 +1,89 @@
+env:
+  browser: true
+  node: true
+  es2021: true
+
+ignorePatterns:
+  - 'public/'
+  - '.cache/'
```

let's set more concrete files for ignore:
```suggestion
  - '**/public/**/*.js'
  - '**/.cache/**/*.js'
```

---

May I ask kindly what the reason is for this action? The 'public' and '.cache' folders are autogenerated and shouldn't be modified.